tests/unit/h/views/account_signup_test.py

```diff
@@ -439,6 +428,20 @@ def test_is_authenticated(matchers, pyramid_request, authenticated_user):
     )
 
 
+def test_encode_idinfo_token(jwt_service):
+    token = encode_idinfo_token(
+        jwt_service, sentinel.provider_unique_id, sentinel.issuer, sentinel.audience
+    )
+
+    jwt_service.encode_symmetric.assert_called_once_with(
+        IDInfo(sentinel.provider_unique_id),
+        expires_in=timedelta(hours=1),
+        issuer=sentinel.issuer,
+        audience=sentinel.audience,
+    )
+    assert token == {"idinfo": jwt_service.encode_symmetric.return_value}
```

Comment:
`encode_idinfo_token()` was previously being covered by the tests for the exception view (above) but the exception view no longer calls this function. The function is still needed because it's imported by other views that want to encode tokens for the views in this file to consume. Added a direct unittest in order to re-add the coverage.